docs/source/ecf.rst

```diff
@@ -1238,6 +1238,14 @@ x_unit
 The x-unit to use in the plot. This can be any unit included in astropy.units.spectral
 (e.g. um, nm, Hz, etc.) but cannot include wavenumber units.
 
+pc_nstep
+''''''''
+The number of time steps used to sample the phase variation when computing the phase curve amplitude and offset.  Defaults to 1000.
+
+pc_stepsize
+'''''''''''
+The sample spacing between independent MCMC steps.  Defaults to 50.
```

Comment:
You should probably add a little bit more detail as to what this parameter is being used for. Something along the lines of `Used to thin the number of MCMC samples used when computing the phase curve amplitude and offset to speed things up. If you want to use all the MCMC samples (and are okay with waiting a while), you can set the parameter to 1`